hudi-common/src/main/java/org/apache/hudi/common/util/collection/RocksDBDAO.java

```diff
@@ -351,6 +354,34 @@ public void delete(String columnFamilyName, String key) {
     return results.stream();
   }
 
+  /**
+   * Return Iterator of key-value pairs from RocksIterator.
+   *
+   * @param columnFamilyName Column Family Name
+   * @param <T>              Type of value stored
+   */
+  public <T extends Serializable> Iterator<T> iterator(String columnFamilyName) {
```

Comment:
@vinothchandar I am working on a lazy iterator to avoid unnecessary memory usage.

Reply:
This is resolved in this PR itself.